NineAnimator/Models/Anime Source/animesaturn.com/AnimeSaturn+Anime.swift

```diff
@@ -0,0 +1,92 @@
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  NineAnimator is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with NineAnimator.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import Foundation
+import SwiftSoup
+
+extension NASourceAnimeSaturn {
+    func anime(from link: AnimeLink) -> NineAnimatorPromise<Anime> {
+        self.requestManager
+            .request(url: link.link, handling: .browsing)
+            .responseString
+            .then {
+                responseContent -> Anime in
+                let bowl = try SwiftSoup.parse(responseContent)
+                let animeTitle = try bowl.select("div.container.anime-title-as.mb-3.w-100 b").text()
+                let animeArtworkUrl = URL(
+                    string: try bowl.select(".cover>img").attr("src")
+                    ) ?? link.image
+                let reconstructedAnimeLink = AnimeLink(
+                    title: animeTitle,
+                    link: link.link,
+                    image: animeArtworkUrl,
+                    source: self
+                )
+                
+                // Obtain the list of episodes
+                let episodes = try bowl.select("div.tab-content div div.episodes-button").reduce(into: [EpisodeLink]()) {
+                    collection, container in
+                    let name =  try container.select("a").text().components(separatedBy: " ")
+                    let episodeName = name[1]
+                    var episodeLink = try container.select("a").attr("href")
+                    episodeLink = episodeLink.replacingOccurrences(of: "'", with: "\'")
+                    if !episodeLink.isEmpty {
+                        collection.append(.init(
+                            identifier: episodeLink,
+                            name: episodeName,
+                            server: NASourceAnimeSaturn.AnimeSaturnStream,
+                            parent: reconstructedAnimeLink
+                            ))
+                    }
+                }
+                
+                // Information
+                let alias = try bowl.select("div.box-trasparente-alternativo.rounded").first()?.text()
+                let animeSynopsis = try bowl.select("#shown-trama").text()
+                //var additionalAttributes = [Anime.AttributeKey: Any]()
+                // Attributes
+                let additionalAttributes = try bowl.select("div.container.shadow.rounded.bg-dark-as-box.mb-3.p-3.w-100.text-white").reduce(into: [Anime.AttributeKey: Any]()) { attributes, entry in
+                    let info = entry.debugDescription.components(separatedBy: "<br>")
```

Comment:
Don't use `debugDescription`. Consider `.html()` or `.outerHtml()` if you really need it.